docs/book/content/servers/index.md

```diff
@@ -1,17 +1,10 @@
-# Adding A Server
-
-To allow using Juniper with the HTTP server of your choice,
```

Comment:
Why did you remove this? It remains accurate.

Reply:
I apologize, I didn't mean to do that

Reply:
Sorry, just a mistake

Reply:
No need to apologize, that's what reviews are for.